Archive.Zip/ZipFileBuilder.cs

```diff
@@ -128,7 +128,7 @@ private void AddFileToZipFile (IFileInfo fileInfo, ZipOutputStream zipOutputStre
         }
         catch (IOException ex)
         {
-          throw new AbortException (
+          throw new IOException (
               string.Format ("Error while copying the data from the file '{0}' to the archive.", fileInfo.FullName),
               ex);
```

Comment:
Does it make sense to include the original `IOException` as InnerException to an `IOException`?

Reply:
Yes, to keep the stacktrace of the original exception.